src/inventory_ui.h

```diff
@@ -0,0 +1,341 @@
+#ifndef INVENTORY_UI_H
+#define INVENTORY_UI_H
+
+#include <memory>
+
+#include "cursesdef.h"
+#include "input.h"
+#include "item_location.h"
+#include "item.h"
```

Comment:
Yes, but if we later want to push inventory functions here (to take them out of `game` class), more files may want to include this. While most of them will want `item.h`, not all will want `item_location`. And if for any reason this header is included within a header (shouldn't happen but can), that `item.h` may creep its way somewhere it shouldn't be.